pkgs/development/python-modules/pandantic/default.nix

```
@@ -0,0 +1,56 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, multiprocess
+, pandas
+, pandas-stubs
+, poetry-core
+, pydantic
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "pandantic";
+  version = "0.3.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "wesselhuising";
+    repo = "pandantic";
+    rev = "f0d749459876a17bd5db53a69ffdfe3bbfc197f5";
```

Upstream tags their releases, see https://github.com/wesselhuising/pandantic/tags

```suggestion
    rev = "rev = "refs/tags/${version}";";
```

---

thanks, I've switched it to use tags :)